proof/records.go

```diff
@@ -55,6 +56,12 @@ func BlockHeaderRecord(header *wire.BlockHeader) tlv.Record {
 	)
 }
 
+func BlockHeightRecord(height *uint32) tlv.Record {
```

Comment:
This commit needs to be earlier, otherwise the previous one doesn't compile.

Reply:
Or just squash this with the previous one, as there are some changes to the same line in here anyway.